src/components/common/ScrollingText/style.module.scss

```diff
@@ -0,0 +1,37 @@
+.overflowing {
+  position: relative;
+
+  .window {
+    margin-right: 2px; // shift cutoff by 2px for antialiased text bleeding through
+    overflow: hidden;
+
+    .slider {
+      display: flex;
+      gap: 2rem;
+    
+      @keyframes marquee {
+        0%   { transform: translateX(0%); }
+        100% { transform: translateX(calc(-100% / 2 - 1rem)); }
+      }
+    
+      &:hover {
+        animation: marquee var(--anim-time, 3s) linear infinite;
```

Comment:
right now the animation resets when your mouse moves off. should it?

you can also keep the animation always on, but set `animation-playing-state` to `paused` until it's hovered

should it play when the mouse hovers on the text or on the card?